src/wasm-lib/kcl/src/lsp/kcl/mod.rs

```diff
@@ -102,12 +102,6 @@ pub struct Backend {
     pub(super) token_map: DashMap<String, TokenStream>,
     /// AST maps.
     pub ast_map: DashMap<String, Node<crate::parsing::ast::types::Program>>,
-    /// Last successful execution.
-    /// This gets set to None when execution errors, or we want to bust the cache on purpose to
-    /// force a re-execution.
-    /// We do not need to manually bust the cache for changed units, that's handled by the cache
-    /// information.
-    pub last_successful_ast_state: Arc<RwLock<Option<OldAstState>>>,
```

Comment:
Did this get folded-in to the rest of the execution caching?

Reply:
oh wtf good catch

Reply:
oh wait this is not my pr sorry for the noise im losing my mind haha

Reply:
I'm just using the static in executioner for caching in the LSP as well as elsewhere. I assume it can be shared between the two since it's the same code being parsed (or if they are different processes, it should still work since there should only be one LSP server per process)